views/frontend/content/index.blade.php

```diff
@@ -5,21 +5,19 @@
 
     <ul>
         @foreach ($apiDocument->data as $discussion)
-            <li>
-                <a href="{{ $url->to('forum')->route('discussion', [
-                    'id' => $discussion->id . (trim($discussion->attributes->slug) ? '-' . $discussion->attributes->slug : '')
-                ]) }}">
-                    {{ $discussion->attributes->title }}
-                </a>
-            </li>
+        <li>
+            <a href="{{ $url->to('forum')->route('discussion', ['id' => $discussion->attributes->idWithSlug]) }}">
+                {{ $discussion->attributes->title }}
+            </a>
+        </li>
         @endforeach
     </ul>
 
     @if (isset($apiDocument->links->prev))
-        <a href="{{ $url->to('forum')->route('index') }}?page={{ $page - 1 }}">&laquo; {{ $translator->trans('core.views.index.previous_page_button') }}</a>
+    <a href="{{ $url->to('forum')->route('index') }}?page={{ $page - 1 }}">&laquo; {{ $translator->trans('core.views.index.previous_page_button') }}</a>
     @endif
 
     @if (isset($apiDocument->links->next))
-        <a href="{{ $url->to('forum')->route('index') }}?page={{ $page + 1 }}">{{ $translator->trans('core.views.index.next_page_button') }} &raquo;</a>
+    <a href="{{ $url->to('forum')->route('index') }}?page={{ $page + 1 }}">{{ $translator->trans('core.views.index.next_page_button') }} &raquo;</a>
     @endif
-</div>
+</div>
```

Comment:
You removed the ending newline.

Also are those indentation changes above an intended change?

Reply:
Fixed, sorry.